request.go

```diff
@@ -70,8 +71,7 @@ func (r *Request) execute(c Client, output interface{}) error {
 	request.Header.Add("X-Auth-UserId", c.cfg.UserUUID)
 	request.Header.Add("X-Auth-Token", c.cfg.APIToken)
 	request.Header.Add("Content-Type", "application/json")
-
-	log.Printf("[DEBUG] Request body: %v", request.Body)
+	log.Infof("Request body: %v", request.Body)
```

Comment:
This should be shown only on debug level.

Reply:
Actually, it is shown only when you set the logLevel = logrus.DebugLevel (or InfoLevel). Anyway I will change the code to meet the above requirements.